server/@types/viewModels/index.d.ts

```diff
@@ -102,4 +102,30 @@ declare module 'viewModels' {
     grade?: string
     source: 'CURIOUS'
   }
+
+  /**
+   * A prisoner's pre-prison work experience, skills, and future work interests.
+   */
+  export interface WorkAndInterests {
+    problemRetrievingData: boolean
+    data?: WorkAndInterestsData
```

Comment:
This `data` field is optional, because its how I'm proposing we know that we called the CIAG API correctly, but there is no data (ie. a 404)